.codesandbox/ci.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packages": ["packages/core"],
+  "buildCommand": "build",
+  "sandboxes": ["filbert-js-demo-7qihx"]
```

Comment:
Q: who owns `filbert-js-demo-7qihx` sandbox ? 
Q: how do I control what goes in sandbox ? e.g in current sandbox, its a CRA. so What do I need to do incase I wanna replace CRA with Nextjs or Preact ..etc. 

@devevignesh

Reply:
A: the example `filbert-js-demo-7qihx` sandbox created from my account
A: you have to create an example sandbox (Nextjs or Preact) from your account. Once it created we need to give that unique string into the config (`ci.json`)

Reply:
@devevignesh thanks for the clearing the doubts. 
I already have a starter kits with a basic example(sandbox) for CRA/Nextjs/Gatsby/Preact ..etc(you can find those under integration section on website). It will be good idea if we use them.

Reply:
sure. let me check and add that

Reply:
@kuldeepkeshwar I have added the rest of the packages and example sandbox